test/runtests.jl

```diff
@@ -2,11 +2,13 @@ using TensorCore
 using LinearAlgebra
 using Test
 
-@testset "Ambiguities" begin
-    @test isempty(detect_ambiguities(TensorCore, Base, Core, LinearAlgebra))
+if VERSION < v"1.5-"
```

Comment:
On 1.6 this could be just `detect_ambiguities(TensorCore)`. The ambiguity detection on 1.6 is better than in previous releases, I've seen it catch quite a few things that were real ambiguities but had somehow been missed by the older versions of Julia. So it would be nice to run the test on 1.6.

Reply:
Will have a look. I guess I wrote that when 1.4 was current & don't recall why.

Reply:
Now passes locally on 1.5 and a recent 1.6, and on travis.